app/Console/Commands/GenerateSitemap.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Route;
+
+class GenerateSitemap extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'sitemap:generate';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate the sitemap.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $routes = ['/' => ['en' => 'en', 'asl' => 'asl', 'fr' => 'fr', 'lsq' => 'lsq']];
+        // once deployed to the server, files have the same modified date, use README as a default last modified date
+        $lastMod = Carbon::createFromTimestamp(filemtime('./README.md'))->toISOString();
+        foreach (Route::getRoutes()->get('GET') as $route) {
+            $routeURI = $route->uri();
+            if (str_contains($routeURI, 'about')) {
```

Comment:
Instead of looking at the route URI I think you can check the route name instead. This is the name we give to the route with `->name()`. So you can get `$route->getName()` and check that value instead as it should be consistent across locales with difference being the local prepended. so something named `about.glossary` would be returned as `en.about.glossary` and so forth if it had be created with `Route::multilingual`

See: https://laravel.com/api/10.x/Illuminate/Routing/Route.html#method_getName

Reply:
That makes sense to use route name instead of uri. One thing I noticed is that glossary is not defined in web.php, so  it is not included when I use getName method. Should we add it as a special case like the main page?

Reply:
There's a require call to bring it in through the web.php at the bottom of the file. I'm surprised it isn't showing up. Shouldn't that call be bringing in all the available routes? We should probably look into this a bit as we may have other things that are defined in separate files in the future.

Reply:
It seemed like we didn't include "about." in glossary route name. I added it and I can include glossary properly.